src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabaseTaskHelper.cs

```diff
@@ -151,6 +153,16 @@ internal static DatabaseInfo DatabasePrototypeToDatabaseInfo(DatabasePrototype p
             return databaseInfo;
         }
 
+        private static string GetBackupDate(DateTime backupDate)
+        {
+            if (backupDate == null
+                || backupDate < minBackupDate)
+            {
+                return SR.NeverBackedUp;
+            }
+            return backupDate.ToString();
```

Comment:
this will return the long datetime format which should be localized. Not a problem for now maybe ...Convert.ToString(dateTime, CultureInfo.CurrentCulture)

Reply:
Hmmm, I wonder if we should do a standard format, for example `YYYY-MM-DD`. @erickangMSFT can you comment? The current way is likely to format by default locale which will annoy / confuse any non-US users.

Reply:
For now I'll leave this as-is and create a tracking task to verify whether it works as expected on machines with non-US preferences.